### PR TITLE
Spatiotemporal guide

### PR DESCRIPTION
**Pull request by josieburgin**
Addition of guidelines for new INSDC spatiotemporal standards coming into effect at the end of May.

### CODE REVIEW

File: faq/spatiotemporal-metadata.rst [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+The `International Sequence Database Collaboration (INSDC) <https://www.insdc.org/>`_ has announced `further details <>`_ on
+the new standards being introduced for reporting spatiotemporal metadata as part of our aim to make sequence data more
+findable, accessible, interoperable and reusable (FAIR). The ENA and its INSDC partners are set to introduce mandatory
+spatio-temporal information for all new samples **by the end of May 2023** unless a valid exemption is declared.
+We feel this is a sensible minimal measure that will improve the utility of INSDC sequence data that all submitters
+should be able to provide and where this can not,
+`suitable reporting options <https://www.insdc.org/submitting-standards/missing-value-reporting/>`_ are available.
+We look forward to seeing the positive impact this has on incoming data.
+
+Implementation plans are shared in the full `INSDC announcement <>`_ with ENA-specific details below.
```
Comment (ihsann1) [inline faq/spatiotemporal-metadata.rst]: URL missing for 'INSDC announcement'

File: faq/spatiotemporal-metadata.rst [hunk 2]
```diff
@@ -0,0 +1,188 @@
+=================================
+Spatiotemporal Metadata Standards
+=================================
+
+The `International Sequence Database Collaboration (INSDC) <https://www.insdc.org/>`_ has announced `further details <>`_ on
```
Comment (ihsann1) [inline faq/spatiotemporal-metadata.rst]: URL missing for 'further details'

File: faq/spatiotemporal-metadata.rst [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+My sample was collected from the Arctic Ocean but the date was not recorded and I only know the year in which it was sampled was sometime in 2019 - how do I report this?
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+The minimum requirement is the name of the ocean/sea (or country) of the collection event and date to the nearest year.
+In this case, as you know the ocean and the year of collection, there is no reason that you can not share these metadata.
+You would report:
+
+**Geographic location (country and/or sea)** = Arctic Ocean
+**collection date**  = 2019
```
Comment (ihsann1) [inline faq/spatiotemporal-metadata.rst:148]: Lines 144 and 145 are both presenting on the same line in preview mode.

File: faq/spatiotemporal-metadata.rst (hunk 1 above)
Comment (josieburgin) [reply faq/spatiotemporal-metadata.rst]: Hi @ihsann1, thank you for this! I think you are reviewing an outdated commit as I added these links this morning and it looks fine to me: 
https://github.com/enasequence/read_docs/pull/94/files

File: faq/spatiotemporal-metadata.rst [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+My sample was collected in the Burgundy region of France at 17:22:05 on the 5th January 2023. How do I format this for submission?
+----------------------------------------------------------------------------------------------------------------------------------
+
+You should report the collection date in the format year-month-day followed by the time in ISO8601 standard format
+including the timezone. You should report the country as required but can also include the region in an additional field.
+For example:
+
+**Geographic location (country and/or sea)** = France
+**Geographic location (region and locality)** = Burgundy
+**collection date**  = 2023-01-05T17:22:05Z+01
```
Comment (ihsann1) [inline faq/spatiotemporal-metadata.rst:174]: Lines 169, 170 and 171 are presenting on the same line in preview mode.

File: faq/spatiotemporal-metadata.rst [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+We recognise there may be valid exemptions for this which are included for missing value reporting. For example,
+you may have collected a control sample from a collection instrument to sequence a negative control. In this case,
+the location of where you collected that control is not applicable to report as it was prepared in a lab but you
+could report the date in which you collected the control sample.
+
+In this case, you would report:
+
+**Geographic location (country and/or sea)** = missing: control sample
+**collection date** = 2020-05-25
+
```
Comment (ihsann1) [inline faq/spatiotemporal-metadata.rst:189]: Lines 184 and 185 are presenting on the same line in preview mode.

File: faq/spatiotemporal-metadata.rst (hunk 1 above)
Comment (ihsann1) [reply faq/spatiotemporal-metadata.rst]: Yes, sorry reloaded to the updated version

**Review by ihsann1 — APPROVED** [review]

**Pull request merged**